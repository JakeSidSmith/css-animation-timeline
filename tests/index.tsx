import * as React from 'react';
import * as renderer from 'react-test-renderer';
import App from '../src/ts/app';

describe('App', () => {

  it('should match snapshot', () => {
    const tree = renderer.create(
      <App />
    );

    expect(tree).toMatchSnapshot();
  })

});
