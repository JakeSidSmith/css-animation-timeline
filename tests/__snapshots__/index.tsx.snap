// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<p>
  Count: 
  <button
    onClick={[Function]}
  >
    Increment
  </button>
</p>
`;
